dash/testing/dash_page.py

```diff
@@ -35,3 +35,15 @@ def redux_state_rqs(self):
         return self.driver.execute_script(
             "return window.store.getState().requestQueue"
         )
+
+    @property
+    def local_storage(self):
+        return self.driver.execute_script(
+            "return JSON.parse(window.localStorage.getItem('local'));"
```

Comment:
not sure about that `local` arg, what are we doing here?

Reply:
this is for `dcc.store` component, convention is set `local` as key for localStorage. that's why it's in dash_page.py as domain relevant helpers

Reply:
good catch, I should either return localStorage or open an arg for that